.github/workflows/trivy-container-scan.yaml

```diff
@@ -0,0 +1,110 @@
+name: trivy-container-image-scan
+
+on:
+  push:
+  pull_request:
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+
+jobs:
+
+  build:
+    name: Scan cactus-corda-4-7-all-in-one-obligation image
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build an image from Dockerfile
+        run: |
+          DOCKER_BUILDKIT=1 docker build ./tools/docker/corda-all-in-one/ -f ./tools/docker/corda-all-in-one/Dockerfile -t cactus-corda-4-7-all-in-one-obligation
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.11.2
+        with:
+          image-ref: 'cactus-corda-4-7-all-in-one-obligation'
+          format: 'table'
+          exit-code: '0'
```

Comment:
Having `exit-code` (as per docs, `Exit code when specified vulnerabilities are found`) as 0 will pass the scan if vulnerabilities are found. So are we addressing those vulnerabilities in another issue and for the time being passing these? (Asking this because if we don't have an issue to track this, then one can fix the vulnerabilities but can forget to change this to a non-zero exit code)

(Same comment for other places where a zero exit-code is returned)

Reply:
Hi @jagpreetsinghsasan, I changed it now to exit-code: 1 so that the scan will not pass when vulnerabilities from high to critical were found.